mkdocs/docs/index.md

```diff
@@ -1,22 +1,156 @@
 # IREE
 
-** * UNDER CONSTRUCTION ** *
+IREE (**I**ntermediate **R**epresentation **E**xecution **E**nvironment[^1]) is
+an [MLIR](https://mlir.llvm.org/)-based end-to-end compiler that lowers Machine
+Learning (ML) models to a unified IR optimized for real-time inference on
+mobile/edge devices against heterogeneous hardware accelerators. IREE also
+provides flexible deployment solutions for its compiled ML models.
 
-## Overview
+## Introduction
```

Comment:
We should just remove this "Introduction" header? It's not really helpful and a bit disruptive to the flow here (as IMO the above sentence is really the introduction).

Reply:
I liked the extra nesting on the table of contents, but agree about disrupting the flow ...

![image](https://user-images.githubusercontent.com/4010439/117221577-6849f180-adbe-11eb-9323-e24c600b96f6.png)

I'll remove it

![image](https://user-images.githubusercontent.com/4010439/117221617-7f88df00-adbe-11eb-9ca5-922306ace079.png)